### ISSUE TITLE
Add support for stringData

### ISSUE THREAD

**Issue by wmedlar** [bitnami-labs/sealed-secrets#104]
The [Secret spec](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.10/#secret-v1-core) allows for non-binary data to be provided in the `stringData` key as a raw string, rather than base64-encoded, for convenience. Most of my secrets are just plain strings, so it'd be awesome to add support for this to kubeseal.

Currently:
* with no `data` key kubeseal panics:
```shell
$ cat secret-no-data.yaml
apiVersion: v1
kind: Secret
metadata:
  name: string-data-secret
  namespace: default
stringData:
  foo: baz

$ kubeseal < secret-no-data.yaml
panic: Secret.data is empty in input Secret, assuming this is an error and aborting

goroutine 1 [running]:
main.main()
	/Users/travis/gopath/src/github.com/bitnami-labs/sealed-secrets/cmd/kubeseal/main.go:233 +0x2a8
```

* with a `data` key kubeseal creates a SealedSecret that decrypts to the key-value pair `foo: bar`
```shell
$ cat secret.yaml
apiVersion: v1
kind: Secret
metadata:
  name: string-data-secret
  namespace: default
data:
  foo: YmFy # bar
stringData:
  foo: baz

$ kubeseal < secret.yaml | kubectl apply -f -
sealedsecret "string-data-secret" created

$ kubectl get secret string-data-secret -o jsonpath="{['data']['foo"]}" | base64 -D
bar
```

In both cases I would expect it to create a SealedSecret that decrypts to the key-value pair `foo: baz`.

**Comment by Renz2018** [comment]
It's very useful!

**Comment by hpurmann** [comment]
I have worked around that issue by [manually base64-encoding the yaml values below the `data` keys](https://github.com/actano/transform-yaml/blob/a6e8965220c86f5dbcf256615ed9391b34f6c252/example/run.sh)

At my company, we are using go template files to specify [from which Vault path to read secrets from](https://github.com/actano/vault-template). A script renders the templates into secrets and seals them. Because `sealed-secrets` don't yet understand `stringData`, we put this manual base64-encoding step in between. It's kind of ugly but works in the meantime.

If of interest, I could share this script.

**Comment by Renz2018** [comment]
@hpurmann
Can you share your code with me?Thanks

**Comment by hpurmann** [comment]
@Renz2018 [I published a gist for you.](https://gist.github.com/hpurmann/fa5bcfc5c3ac392cf8ef6a4d9a7f48b6) Hope that helps.

**Comment by hpurmann** [comment]
@Renz2018 We re-wrote this in Go and release it as a binary: https://github.com/actano/helm-sealed-secrets

**Comment by dmitrievav** [comment]
`cat secret.yaml | k8s-secret-to-base64 | kubeseal`

where k8s-secret-to-base64 is:

```
#!/usr/bin/env python
# -*- encoding: utf-8 -*-

from __future__ import print_function
from __future__ import division
import sys
import yaml
import base64

if __name__ == '__main__':
  try:
    data = yaml.safe_load(sys.stdin.read())
  except yaml.YAMLError as exc:
      print(exc)
  if "stringData" in data:
    for key, value in data["stringData"].items():
      data["stringData"][key] = base64.b64encode(data["stringData"][key])
    data["data"] = data.pop("stringData")

  yaml.dump(data, sys.stdout, default_flow_style=False, default_style='"')
```

**Comment by WhatTheFar** [comment]
I have created a simple workaround script for encoding secret's `stringData` to base64 `data`.
The tool is written in Golang, so it can be downloaded as binary for the CI/CD.

Here is the repository, https://github.com/WhatTheFar/kube-secret-encode

The example usage with `kubeseal` would be :
```bash
cat your-secret.yaml | kube-secret-encode | kubeseal --format yaml > your-secret.sealed.yaml
```

Hopefully, this would be helpful.

**Comment by mkmik** [comment]
Implemented and released in v0.8.2. Sorry I forgot to tag and close this issue.
